crates/lair_keystore/src/bin/lair-keystore/main.rs

```diff
@@ -5,15 +5,33 @@
 use structopt::StructOpt;
 use tracing::*;
 
+static LAIR_KEYSTORE_ABOUT: &str = r#"A secure storage system for Holochain cryptographic keys and secrets.
+
+- one `lair-keystore` per `holochain`
```

Comment:
I mention this because otherwise, if two instances use the same kitsune cert which apparently comes from lair and install the same DNA, then their networking will freak when they try to connect to each other
```
WARN gossip_loop: kitsune_p2p::spawn::actor::gossip: msg="gossip loop error" e=KitsuneError(KitsuneError(Other(OtherError("refusing outgoing connection to node with same cert"))))
Apr 26 18:14:33.136  WARN gossip_loop:process_next_gossip: kitsune_p2p_proxy::tx2: refusing outgoing connection to node with same cert
```
I wrote about it here: https://forum.holochain.org/t/avoiding-networking-issue-for-multiple-nodes-on-1-device/5688